lib/nats/io/client.rb

```diff
@@ -135,6 +135,8 @@ def default_reloader
       # Re-establish connection in a new process after forking to start new threads.
       def after_fork
         INSTANCES.each do |client|
+          next if client.closed?
```

Comment:
this one related to? https://github.com/nats-io/nats-pure.rb/issues/153

Reply:
No, I think, it's a different one; @vankiru is on it